content/applications/hr/time_off.rst

```diff
@@ -0,0 +1,634 @@
+  more fields appear to specify exactly when the accrual renews.
+
+.. example::
+   If the employee should accrue one vacation day for every week worked, the :guilabel:`Rate` is set
+   to `1`, and the :guilabel:`Frequency` entry is set to `Frequency (Weekly) on (Friday)`. Only the
+   :guilabel:`Frequency` and :guilabel:`Weekday` fields appear.
+
+   If the employee should accrue ten vacation days each year, and they receive these days every year
+   on the first of January, the :guilabel:`Rate` is set to `10`, and the :guilabel:`Frequency` entry
+   is set to `Frequency (Yearly) on the (1) of (January)`. The :guilabel:`Frequency`,
+   :guilabel:`Date`, and :guilabel:`Month` fields appear.
+
+- :guilabel:`Limit to`: enter a maximum amount of days the employee can accrue with this plan.
+- :guilabel:`At the end of the calendar year, unused accruals will be`: select from the drop-down
+  menu how unused time off is handled. Options are either :guilabel:`Transferred to the next year`,
+  which rolls over unused time to the next calendar year, or :guilabel:`Lost`, which means any
+  unused time off is gone.
+
+.. image:: time_off/new-level.png
+   :align: center
+   :alt: A level form with all the data filled out.
+
+When the form is filled out, click :guilabel:`Save & Close` to save the form and close the pop-up,
+or :guilabel:`Save & New` to save the form and create a new rule. Add as many levels as desired.
+
+.. image:: time_off/accrual-plan-form.png
+   :align: center
+   :alt: An accrual form with all the entries filled out.
+
+.. _time_off/public-holidays:
+
+Public holidays
+---------------
+
+Most countries have public or national holidays, and some companies may have specific days they
+are closed and give extra days as holidays. It is important to configure these days in Odoo, so that
+employees are aware of the days they have off, and do not request time off on days that are already
+set as a public holiday (non-working days).
+
+Create a new public holiday
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To create a public holiday, navigate to :menuselection:`Time Off application --> Configuration -->
+Public Holidays`. All currently configured public holidays appear in a list view. Click the
+:guilabel:`Create` button and a new line appears at the bottom of the list. Enter the following
+information:
+
+- :guilabel:`Name`: enter the name of the holiday.
+- :guilabel:`Company`: if in a multi-company database, the current company populates this field by
+  default. It is not possible to edit this field.
+- :guilabel:`Start Date`: using the date and time picker, select the date and time that the holiday
+  starts. By default, this field is configured for the current date, with a time of 8:00:00 (8 AM).
+- :guilabel:`End Date`: using the date and time picker, select the date and time that the holiday
+  ends. By default, this field is configured for the current date, with a time of 17:00:00 (5 PM).
```

Comment:
And for me, now, it's showing 2 and 11 =D. I am checking with the experts as to how they are getting these hours, since it doesn't seem to be based off of the company location, local time, etc.